.github/workflows/build.yaml

```diff
@@ -11,11 +11,13 @@ jobs:
   build:
     runs-on: ubuntu-latest
     strategy:
+      fail-fast: false
       matrix:
         env:
           - ALPINE_VERSION=3.17 ROS_DISTRO=noetic ROS_DISTRIBUTION_TYPE=ros1
           - ALPINE_VERSION=3.17 ROS_DISTRO=humble ROS_DISTRIBUTION_TYPE=ros2
           - ALPINE_VERSION=3.20 ROS_DISTRO=noetic ROS_DISTRIBUTION_TYPE=ros1
+          - ALPINE_VERSION=3.20 ROS_DISTRO=noetic ROS_DISTRIBUTION_TYPE=ros1 SPLIT_DEV=yes
```

Comment:
Will we keep both alpine 3.20 noetic jobs or was it just for development?

Reply:
Our Alpine ROS repository will split dev packages, but users still can keep them not splitted when building their packages using ros-abuild-docker image

Reply:
The `ros-abuild:3.20-noetic` image is pushed twice but I guess it is not a problem.

Reply:
I'm gonna revert the matrix and test split-dev mode in each matrixed job

Reply:
Fixed CI

Reply:
Thank you!

Reply:
> Our Alpine ROS repository will split dev packages, but users still can keep them not splitted when building their packages using ros-abuild-docker image

I noticed this is wrong.
As `ros-noetic-catkin` doesn't contain dev files, catkin packages on v3.20 must be always built by split-dev mode.

Reply:
Fixed to test only split-dev mode on v3.20

Reply:
Updated docker image to automatically enable split-dev mode on 3.20